core/src/main/resources/org/apache/spark/ui/static/additional-metrics.js

```diff
@@ -26,17 +29,31 @@ $(function() {
         // Switch the class of the arrow from open to closed.
         $(this).find('.expand-additional-metrics-arrow').toggleClass('arrow-open');
         $(this).find('.expand-additional-metrics-arrow').toggleClass('arrow-closed');
+
+        window.localStorage.setItem("expand-additional-metrics", "" + status);
     });
 
+    if (window.localStorage.getItem("expand-additional-metrics") == "true") {
```

Comment:
minor, but we probably don't need this check; if it's already false then it doesn't do anything if we set it to false again. Same in L74 and in other files.